src/ReactDragListView.jsx

```diff
@@ -24,6 +25,7 @@ class ReactDragListView extends Component {
 
   static defaultProps = {
     nodeSelector: DEFAULT_NODE_SELECTOR,
+    ignoreSelector: undefined,
```

Comment:
Recommended to be consistent with the default type of handleSelector

Reply:
fixed